service/src/main/java/bio/terra/pipelines/service/PipelinesService.java

```diff
@@ -101,6 +113,7 @@ public List<String> validateRequiredInputs(
       List<PipelineInputDefinition> inputDefinitions, Map<String, Object> inputsMap) {
     ArrayList<String> errorMessages = new ArrayList<>();
     inputDefinitions.stream()
+        .filter(PipelineInputDefinition::getUserProvided)
```

Comment:
alternatively, rather than filtering here, we could make different repository methods (i.e. queries) to grab the user or service provided inputs

Reply:
too complicated for now based on the way we're pulling the input definitions via the pipeline entity